src/python/zquantum/core/circuit/_gateset.py

```diff
@@ -19,26 +19,4 @@
 COMMON_GATES = ['X', 'Y', 'Z', 'H', 'S', 'T', 'PHASE', 'Rx', 'Ry', 'Rz', 
                 'CNOT', 'CZ', 'CPHASE', 'SWAP']
 UNIQUE_GATES = ['ZXZ', 'RH', 'XX', 'YY', 'ZZ', 'U1ex', 'Da', 'Db', 'MEASURE', 'BARRIER'] # gates unique to specific packages
-ALL_GATES = COMMON_GATES + UNIQUE_GATES
-
-
-# All gates natively supported in pyquil 
-# (generated from pyquil.gates.__dict__['__all__'])
-PYQUIL_GATES = ['I', 'X', 'Y', 'Z', 'H', 'S', 'T', 'PHASE', 'RX', 'RY', 'RZ', 'CZ', 
-                'CNOT', 'CCNOT', 'CPHASE00', 'CPHASE01', 'CPHASE10', 'CPHASE', 'SWAP', 
-                'CSWAP', 'ISWAP', 'PSWAP', 'WAIT', 'RESET', 'NOP', 'HALT', 'MEASURE', 
-                'TRUE', 'FALSE', 'NOT', 'AND', 'OR', 'MOVE', 'EXCHANGE', 'IOR', 'XOR', 
-                'NEG', 'ADD', 'SUB', 'MUL', 'DIV', 'EQ', 'GT', 'GE', 'LE', 'LT', 'LOAD', 
-                'STORE', 'CONVERT', 'Gate']
-
-# All gates natively supported in cirq
-CIRQ_GATES = ['PhasedXPowGate', 'CNOT', 'CNotPowGate', 'CZ', 'CZPowGate', 'H', 'HPowGate', 
-              'ISWAP', 'ISwapPowGate', 'measure', 'measure_each', 'MeasurementGate', 'XPowGate', 
-              'YPowGate', 'ZPowGate', 'Rx', 'Ry', 'Rz', 'S', 'SWAP', 'SwapPowGate', 'T', 'X', 'Y', 
-              'Z', 'XX', 'XXPowGate', 'YY', 'YYPowGate', 'ZZ', 'ZZPowGate', 'PhasedXPowGate',
-              'CCX', 'CCXPowGate', 'CCZ', 'CCZPowGate', 'CSWAP', 'CSwapGate', 'FREDKIN', 'TOFFOLI']
-                        
-# All gates natively support in qiskit
-QISKIT_GATES = ['ubase', 'u2', 'h', 'cxbase', 'cx', 'u1', 't', 'ccx', 'cswap', 'cx_base', 's', 'cy', 
-                'cz', 'swap', 'iden', 'sdg', 'tdg', 'u0', 'u3', 'u_base', 'x', 'y', 'z', 'rx', 'ry', 
-                'rz', 'cu1', 'ch', 'crz', 'cu3', 'rzz', 'measure', 'barrier']
```

Comment:
Do we not use these anywhere else?

Reply:
Anywhere at all. Seems like some legacy code to me.

Reply:
Alrighty then - time to get rid of it :D